src/test/java/org/jvnet/hudson/plugins/platformlabeler/NodeLabelCacheTest.java

```diff
@@ -1,7 +1,11 @@
 package org.jvnet.hudson.plugins.platformlabeler;
 
-import static org.hamcrest.Matchers.*;
-import static org.junit.Assert.*;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.everyItem;
```

Comment:
If you're determined to expand the wildcard imports of the assert convenience methods, please remove unused imports.  I prefer wildcard static imports for those convenience methods, but I'm fine with the decision to replace them with direct imports, so long as we don't add unused imports in the process.  As far as I can tell, everyitem is an unused import.
```suggestion
```

Reply:
reverted back to wildcard imports